jbmc/src/java_bytecode/java_object_factory.cpp

```diff
@@ -899,26 +899,22 @@ void java_object_factoryt::gen_nondet_struct_init(
 /// \param expr The expression we want to make an assumption on
 /// \param type The type of the expression
 /// \param location Source location associated with the expression
-/// \param [out] symbol_table Symbol table, a new auxiliary symbol will be added
+/// \param allocate_local_symbol A function to generate a new local symbol
+///   and add it to the symbol table
```

Comment:
:pick: Maybe say "why" in commit message

Reply:
Done